src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -2,72 +2,55 @@
 title: Saved Searches
 sidebar_order: 20
 keywords: ["saved search", "premade searches"]
-description: "Learn more about recommended, pinned, and saved searches."
+description: "Learn more about default, recommended, and saved searches."
 ---
 
-In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+Sentry allows you to save searches that you find most useful, either for yourself or for your entire organization.
 
-## Recommended Searches
+## Saved Searches
```

Comment:
We don't need this heading since the whole page is about saved searches. We can just go straight from the intro sentence into telling people how to access it.
```suggestion
```